src/components/Component7.js

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+function Component7() {
```

Comment:
Try to use consistent line spacing between import statements and code (1 empty line)